### PR TITLE
[Fix] Centreon login ignore insecure

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/39277

## Description
* Packified Centreon
* Fixed an issue in `login()` function where the `requests.post` didn't pass `USE_SSL`.

### CODE REVIEW

**Comment by guykeller** [comment]
@DeanArbel Ping me once you fix the build and I'll review the final product

**Comment by DeanArbel** [comment]
@guykeller Do note that the validation is failing because of the skipped tests. This won't change, since we don't have an instance.

File: Packs/Centreon/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Centreon
+- Fixed an issue where **login** ignored **Trust any certificate (not secure)** configuration.
+- Updated docker image from `2.7.18.20958` to `2.7.18.22912`.
```
Comment (bakatzir) [inline Packs/Centreon/ReleaseNotes/1_0_2.md:5]:
```suggestion
- Fixed an issue where the **Trust any certificate (not secure)** configuration was ignored.
- Updated the docker image to: *demisto/python:2.7.18.20958*.
```

File: Packs/Centreon/Integrations/Centreon/Centreon.yml [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+      description: Service description
+      type: string
+    - contextPath: Centreon.Service.State
+      description: Service state
+    description: All the monitoring information regarding services
+  runonce: false
+  dockerimage: demisto/python:2.7.18.22912
+tests:
+- Centreon-Test-Playbook
+fromversion: 5.5.0
```
Comment (bakatzir) [inline Packs/Centreon/Integrations/Centreon/Centreon.yml:159]: it was fromversion 5.0.0 in the unified yml

**Review by bakatzir — COMMENTED** [review]
I guess we do not have an instance correct?
in any case, please generate the README, even without examples.

File: Packs/Centreon/Integrations/Centreon/Centreon.yml (hunk 2 above)
Comment (DeanArbel) [reply Packs/Centreon/Integrations/Centreon/Centreon.yml:159]: It was, but we no longer support 5.0.0

Comment (DeanArbel) [reply Packs/Centreon/Integrations/Centreon/Centreon.yml:159]:
```suggestion
fromversion: 5.0.0
```

**Review by bakatzir — APPROVED** [review]

**Comment by DeanArbel** [comment]
We've confirmed changes work, so going ahead with the merge.

**Pull request merged**